droptools-example/build.gradle

```diff
@@ -1,24 +1,32 @@
-group = "com.bendb.droptools"
-description = "blah"
-
 buildscript {
     repositories {
+        mavenCentral()
         maven {
             url "https://plugins.gradle.org/m2/"
         }
     }
 
     dependencies {
-        classpath "nu.studer:gradle-jooq-plugin:3.0.3"
-        classpath "com.github.jengelman.gradle.plugins:shadow:5.0.0"
+        classpath "com.github.johnrengelman:shadow:8.1.1"
+        classpath "org.postgresql:postgresql:$postgresPluginVer"
+        classpath group: 'org.jooq', name: 'jooq', version: jooqVersion
+        classpath group: 'org.jooq', name: 'jooq-meta', version: jooqVersion
+        classpath group: 'org.jooq', name: 'jooq-codegen', version: jooqVersion
     }
 }
 
+plugins {
+    id "com.github.ben-manes.versions" version "0.46.0"
+}
+
+group = "com.bendb.droptools"
+description = "blah"
+
 apply plugin: "application"
-apply plugin: "nu.studer.jooq"
+//apply plugin: "nu.studer.jooq"
```

Comment:
Again (and this goes for the rest of the files here) let's just delete code rather than leave it commented-out.